packages/legend-studio-app/src/index.tsx

```diff
@@ -16,12 +16,13 @@
 
 import { Studio } from '@finos/legend-studio';
 import { DSLText_Preset } from '@finos/legend-studio-preset-dsl-text';
+import { EFJSONSchema_Preset } from '@finos/legend-studio-preset-external-format-json-schema';
 import studioConfig from '../studio.config';
 import './index.scss';
 
 Studio.create()
   .setup({ baseUrl: studioConfig.baseUrl })
-  .withPresets([new DSLText_Preset()])
+  .withPresets([new DSLText_Preset(), new EFJSONSchema_Preset()])
```

Comment:
oh how I hate the fact this must be a separate module in Studio 😡 😡 😡

Reply:
okay sir